src/lib/markbind/src/parser.js

```diff
@@ -94,6 +94,15 @@ function Parser(options) {
  */
 function extractIncludeVariables(includeElement, contextVariables) {
   const includedVariables = { ...contextVariables };
+  Object.keys(includeElement.attribs).forEach((attribute) => {
+    if (!attribute.startsWith('var-')) {
+      return;
+    }
+    const variableName = attribute.replace(/^var-/, '');
+    if (!includedVariables[variableName]) {
+      includedVariables[variableName] = includeElement.attribs[attribute];
```

Comment:
Is it possible to run through the variable through nunjucks?

Reply:
Hmm I think at this point the variable would have already been ran through nunjucks based on the preprocessing

Reply:
I tested it with `<include src="foo.md" var-x="bat {{ color }}" />` where color is a global var and it works correctly

Reply:
You are right, I stand corrected.